src/Config.php

```diff
@@ -118,9 +118,11 @@ public static function canCreate()
     public function canViewItem()
     {
         if (
-            isset($this->fields['context']) &&
-            ($this->fields['context'] == 'core' ||
-            Plugin::isPluginActive($this->fields['context']))
+            isset($this->fields['context'])
+            && (
+                in_array($this->fields['context'], ['core', 'inventory'], true) // GLPI config contexts
+                || Plugin::isPluginActive($this->fields['context'])
+            )
```

Comment:
@cedric-anne, thanks for the speedy reply.

I've tested this change and it only makes it visible, an api call to `http://127.0.0.1/apirest.php/Config?range=0-1000` still doesn't incude the inventory settings.

also editing still gives permission denied

Reply:
correction to my last, i can edit the setting (opps, needed HTTP/PUT), however fetching all settings with url `http://127.0.0.1/apirest.php/Config?range=0-1000` doesn't return inventory items

Reply:
I added a commit that should fix this.